src/web/backend/nextcloud/api.py

```diff
@@ -36,32 +54,49 @@ class UserViewSet(NextCloudMixin,
     def get(self, username=None):
```

Comment:
Please split this into two functions - it is basically two function in one, depends on whether `username` is not `None`.

Reply:
Agree, split it on two viewsets for user list actions and single user actions.